### ISSUE TITLE
Add documentation for correctly setting CA certificates.

### ISSUE THREAD

**Issue by gamuniz** [ansible/tower-cli#194]
There is very little documentation on the proper setup for a CA generated certification. I'm also not sure if the verification is working correctly. 

Using a external certificate is giving this error:

```
Error: Could not establish a secure connection. Please add the server to your certificate authority.

```

**Comment by otdouglasm** [comment]
This issue was filed from my ticket with Ansible support, request # 9277, and @michelleperz asked me to add this request.  In the interim, can you please research and provide a procedure to resolve the error? Disabling SSL verification is only a temporary workaround, and I need to be able to use your product securely.

**Comment by samdoran** [comment]
On RHEL/CentOS you can add new CAs to the trust store by putting them in `/etc/pki/ca-trust/source/anchors/` and running `update-ca-trust`.

**Comment by jangsutsr** [comment]
@gamuniz Custom certificate support is implemented, will come out with 3.0.1. You can find documentation of implementing custom CA cert in `docs/CONFIG_CMD_OPTIONS.md`.
